src/components/chatbox/welcome-message.module.css

```diff
@@ -50,45 +42,61 @@
 	}
 
 	height: 100%;
+	padding: 16px 108px;
```

Comment:
Had to use a slightly smaller L/R padding here since the spec shows (123px) the command bar at like 852px and in prod the command bar with is 820px.